src/main/java/seedu/mark/storage/JsonAdaptedReminder.java

```diff
@@ -11,20 +18,24 @@
 public class JsonAdaptedReminder {
 
     public static final String MISSING_FIELD_MESSAGE_FORMAT = "Reminder's %s field is missing!";
+    private static final String DATE_FORMATTER = "dd/MM/yyyy HHmm";
+    protected static final String MESSAGE_INVALID_TIME_FORMAT =
+            "Invalid time format! Please use the following format: " + DATE_FORMATTER;
```

Comment:
Not very important, but easier for user to understand (when the whole json file is read and errors appear).
```suggestion
            "Reminder time has the wrong format! Please use the following format: " + DATE_FORMATTER;
```